idseq_pipeline/commands/common.py

```diff
@@ -95,6 +97,11 @@ def count_reads(self, func_name, before_filename, before_filetype, after_filenam
             self.data = new_data
         self.data.append({'task': func_name, 'reads_before': records_before, 'reads_after': records_after})
 
+    def set_remaining_reads(self):
+        last_entry = self.data[-1] or {}
+        remaining = last_entry.get('reads_after', 0)
+        self.data.append({'remaining_reads': remaining})
```

Comment:
If here we append, we should search backgward in get_remaining_reads above :)  in case multiple updates

however unlikely

minor point

don't care :)